terminology-guides/spanish-terminology.md

```diff
@@ -4,39 +4,50 @@
 | --- | --- | --- |
 | Amounts | Cantidades | |
 | Blockchain | Blockchain o Cadena de Bloques? | Mantiene su forma original al ser una nueva tecnología. En algunas ocasiones el término Cadena de bloques, incluso dentro de Bitcoin.org [se define como Cadena de bloques](https://bitcoin.org/es/vocabulario#cadena-de-bloques). |
+| Contributors | Contribuyentes o colaboradores | El término "contribuidores" es la definición literal pero en su uso es poco extendida. |
+| Custom decorations | Decoraciones personalizadas | |
 | Daemon | Daemon |  |
+| Exploit | Vulnerabilidad / Comprometido | |
+| File | Archivo | Es mejor conocido dentro de la comunidad hispanohablante |
 | Flag | Marcador | Flag es un marcador. En algunas traducciones en la GUI se utiliza la palabra opción. Discusiones abiertas en #799 y #827 |
 | Forum Funding System - FFS | Sistema de Financiamiento Colectivo - SFC | |
 | Hash | Hash | --- |
+| Hash Rate | Poder de procesamiento o Poder de Hash | [Bitcoin.org "hash rate"](https://bitcoin.org/en/vocabulary#hash-rate) medida con la que se mide al poder de procesamiento. |
 | Keys | Claves | Se discutió "llaves", pero la comunidad Bitcoin ya utiliza la palabra "claves" |
+| Log categories | Categorias de registros | |
+| Mainnet | Red principal | |
 | Mine | Minar | |
 | Miner | Minero o Minador| |
+| Mining | Minería | |
 | Mnemonic seed | Semilla mnemónica | Mnemónica,co [RAE](http://dle.rae.es/?id=PRbUjYS) "Mnemo~ ⇒ Mnemósine, diosa griega de la memoria + técnica, que signifícan, técnicas para ayudar la memoria o, sencillamente, técnicas de memorización." Fuente [Mnemotecnia](https://www.mnemotecnia.es/inicio). |
 | Open-source | Código abierto | |
 | Paper wallet | Monedero en papel | |
 | Pool | Pool | Se queda "pool" y no "piscina", ya que pool hace referencia a esfuerzo o valor compartido |
+| Pool Mining | Minería en pool | |
+| Public view key | Clave Pública de visualización | |
+| Public spend key | Clave de gasto público / Clave pública de gasto | Open discussion |
 | Receiver | Receptor | |
+| Remote Node Port | Puerto de Nodo Remoto | |
+| Rescan wallet balance | Reescanear el balance del monedero / Volver a escanear el saldo del monedero | Open to discussion |
 | Ring signature | Firma de círculo | Se mantiene como firma de círculo por su uso en wikipedia |
+| Run | Ejecutar, Correr, Iniciar | Consistencia en términos ya utilizados y generalizados en diferentes Sistemas Operativos. [Término "Run" en Microsoft](https://www.microsoft.com/en-us/language/Search?&searchTerm=run&langID=Spanish&Source=true&productid=All) |
+| Secret spend key | Clave secreta de gasto | |
+| Secret view key | Clave secreta de visualización | |
```

Comment:
The same term is already being used in another word, in this case it is **view key**, the *secret* (or *public*, *private*) part it's more easy and not part of the crypto-technicalities. What I mean is that we can just leave Spend Key, View Key, Private Key as the listed words for the terminology, what do you think? :)

Reply:
I understand, :+1: